include/envoy/upstream/cluster_manager.h

```diff
@@ -149,12 +150,11 @@ class ClusterManagerFactory {
   /**
    * Allocate a cluster manager from configuration JSON.
    */
-  virtual ClusterManagerPtr clusterManagerFromJson(const Json::Object& config, Stats::Store& stats,
-                                                   ThreadLocal::Instance& tls,
-                                                   Runtime::Loader& runtime,
-                                                   Runtime::RandomGenerator& random,
-                                                   const LocalInfo::LocalInfo& local_info,
-                                                   AccessLog::AccessLogManager& log_manager) PURE;
+  virtual ClusterManagerPtr
+  clusterManagerFromJson(const Json::Object& config, const envoy::api::v2::Bootstrap& bootstrap,
```

Comment:
Will add this in a later PR, once we grow the bootstrap proto to encompass the full cluster manager, so yeah, this is the plan, but for a minimal CDS/EDS end-to-end capability this isn't needed yet.